### ISSUE TITLE
x/pkgsite: package removal request for github.com/4rcode

### ISSUE THREAD

**Issue by sam-4rcode** [golang/go#49002]
### What is the path of the package that you would like to have removed?

all modules under github.com/4rcode
 
### Are you the owner of this package?

Yes

### What is the reason that you could not retract this package instead?

messed up the modules while renaming/moving/making private/etc...

Thanks!

**Comment by jba** [comment]
Done.

**Comment by sam-4rcode** [comment]
Hi @jba ,

I've recently noticed that google doesn't just cache the contents of the git repos, but also the hash sums of each version in the sumdb mirror. I found online a workaround to disable the use of both the proxy and the sumdb via GOPROXY and GOSUMDB.
However I'd like to know if there's an official way to get rid completely of any information about a repo from proxy.golang.org, sum.golang.org, and index.golang.org.

I genuinely didn't know that since go version 1.13 the proxy was actively collecting data.

Regards
  Sam
